cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -42,11 +43,16 @@ Type = "vault"
 Protocol = "http"
 Host = "localhost"
 Port = 8200
-HealthCheckPath = "v1/sys/health"
-TokenPath = "res/resp-init.json"
-CertPath = ""
-CACertPath = ""
-SNIS = [""]
+Path = "/v1/secret/edgex/edgex-security-proxy-setup/"
+TokenFile = "/tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json"
```

Comment:
Since we change this name from `TokenPath ` to `TokenFile`, please make the corresponding environment variable name change in snap.

Reply:
Will do! Thansk!